theme/beevibrant/templates/landing.html

```diff
@@ -3,15 +3,9 @@
 {% block content %}
 {{ super() }}
 {% if page %}
-<section class="tg-sectionspace tg-haslayout" id="euroscipy-page">
```

Comment:
keep this for later

Reply:
one of the H's didn't work with one of the styles applied from those classes

Reply:
I don't understand the "later" Guillaume was referring to? Is that related to the schedule ?

Reply:
I think, he noticed, that I have removed a bunch of classes from the html to simplify things. For him that change looked a bit out of scope, but I have clarified that these classes are not compatible with changes I made.